legal-api/src/legal_api/reports/business_document.py

```diff
@@ -553,6 +553,12 @@ def _get_legal_type_description(legal_type: str) -> str:
                 'SP': 'Dissolution Application',
                 'GP': 'Dissolution Application'
             },
+            'involuntary': {
+                'BC': 'Involuntary Dissolution',
+                'BEN': 'Involuntary Dissolution',
+                'ULC': 'Involuntary Dissolution',
+                'CC': 'Involuntary Dissolution',
+            },
```

Comment:
Do you need to add more for other legal types?

Reply:
Thx, Hongjing.

Yes, I found we have a lot of eligible types [here](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/involuntary_dissolution.py#L28C5-L39C6).  After confirming with Argus, it seems that these four types are fine.

Let me know if there are other types that need to be added.

Reply:
Good to know, thanks! Resolved.